umh.docs.umh.app/content/en/docs/features/umh-lite.md

```diff
@@ -0,0 +1,91 @@
+you will not install Grafana or Kafka, and you will not have a Unified
+Namespace. However, you will be able to monitor the connection health of your
+network and spot errors with a single glance.
+
+## What can I do with it?
+
+### Differences between UMH Lite and Classic
+
+The UMH Lite does not offer all features of the UMH Classic. In the table below,
+you can find a comparison between both, and check if the UMH Lite fulfills your
+needs.
+
+| Feature                   | Classic | Lite |
+|---------------------------|:-------:|:----:|
+| Historian                 |    ✓    |      |
+| Analytics                 |    ✓    |      |
+| Data Visualization        |    ✓    |      |
+| UNS                       |    ✓    |      |
+| Kafka and MQTT            |    ✓    |      |
+| Alerting                  |    ✓    |      |
+| Connectivity: OPC UA      |    ✓    |  ✓   |
+| Connectivity: Node-RED    |    ✓    |      |
+| Network Monitoring        |    ✓    |  ✓   |
+| UMH Data Model v1         |    ✓    |  ✓   |
+| Tag Browser for your UNF  |    ✓    |      |
+
+### Connect devices and add protocol converters
+
+You can connect external devices like a PLC with an OPC UA server to a running
+UMH Lite instance and contextualize the data from it with a [protocol
+converter](https://umh.docs.umh.app/docs/features/connectivity/benthos-umh/).
+For contextualization, you have to use the [UMH Data Model
+v1](https://umh.docs.umh.app/docs/datamodel/messages/).
+
+### Send data to your own infrastructure
+
+All the data that your instance is gathering is sent to your own data
+infrastructure. You can configure a target MQTT broker in the instance settings
+by clicking on it in the Management Console.
+
+### Monitor your network health
+
+By using the UMH Lite in conjunction with the Management Console, you can spot
+errors in the network. If a connection is faulty, the Management Console will
+mark it.
+
+### Convert to a UMH Classic
+
+If you notice at any time that the UMH Lite is not enough anymore and you need features from the UMH Classic, you can convert it to a UMH Classic in
+the Management Console. This will not affect your configured devices and
+protocol converters: Their data will still be sent to your MQTT broker and
+additionally will be available in your new Unified Namespace and database.
+
+## How can I use it?
+
+To add a new UMH Lite instance, simply follow the regular 
+[installation process](https://umh.docs.umh.app/docs/getstarted/installation/) and select UMH Lite instead of UMH Classic. You can follow the next steps in
+the linked guide to learn how to connect devices and add a protocol converter.
+
+## What are the limitations?
+
+The UMH Lite is a very basic version and only offers you the gathering and
+contextualization of your data as well as the monitoring of the network. Other
+features like a historian, data visualization, and a Unified Namespace are only
+available by using the UMH Classic.
+
```

Comment:
ADD: The convert to UMH Classic requires an SSH connection.

Reply:
Add: only the protocol converters that were configured pre conversion will continue to sent data to the MQTT broker, new ones will not. But u can add an external connection for the whole instance.

Reply:
Note, not necessarily add to docu: The MQTT setting can be manually deleted from the `mgmtcompanion-config`. When this is done, ALL converters, both existing and future, will only use the internal Kafka broker as their output.
Requirements